tests/commoncode/test_paths.py

```diff
@@ -83,12 +86,18 @@ def test_safe_path_posix_style_absolute_trailing_slash(self):
 
     def test_safe_path_posix_style_french_char(self):
         test = paths.safe_path('/includes/webform.compon\xc3nts.inc/')
-        expected = 'includes/webform.componAnts.inc'
+        if py2:
+           expected = 'includes/webform.componAnts.inc'
+        else:
+           expected = 'includes/webform.compon_nts.inc'
```

Comment:
Why do not we get the same transliteration on Python 2 and 3?

Reply:
According to console, it requires.